spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AboutControllerTests.java

```diff
@@ -398,6 +400,8 @@ public void testAbout() throws Exception {
 					.andExpect(jsonPath("$.runtimeEnvironment.appDeployer.deployerName", is("skipper server")))
 					.andExpect(jsonPath("$.monitoringDashboardInfo.url", is("http://localhost:3001")))
 					.andExpect(jsonPath("$.monitoringDashboardInfo.dashboardType", is("GRAFANA")))
+					.andExpect(jsonPath("$.gitAndBuildInfo.git.commit").exists() )
+					.andExpect(jsonPath("$.gitAndBuildInfo.build.version").exists() )
 					.andExpect(jsonPath("$.monitoringDashboardInfo.refreshInterval", is(30)));
```

Comment:
I think it would be good to add tests for the following cases:
- when no contributor info then no git and build info in the results (ie `doesNotExists()`)
- when only git contributor then only git info
- when only build contributor then only build info

Reply:
Can we discuss this part?? 
- The git contributor will be missing if this code doesn’t have any `.git` folder or the `git-commit-id-maven-plugin` plugin is not present, and due this project has both, probably doesn’t make sense to have something that evaluates to none existing.
- The same for the build info.

**NOTE**: The extreme case will be when we manual delete both and test for that.... but they will be re-generated regardless.

WDYT??

Reply:
The tests are mostly fail safes to make sure if actuator is excluded or the user disables eitehr of the contributors via props that we don't 💣 . 

I don't mind adding these other cases during a polish commit either though. Let's stick w/ a single test now and I will do that. No worries.